configTools/src/main/scala/com/gu/janus/VerifyJanusData.scala

```diff
@@ -0,0 +1,43 @@
+package com.gu.janus
+
+import java.io.File
+
+import com.gu.janus.JanusConfig.JanusConfigurationException
+
+import scala.Console.{GREEN, RED, RESET}
+import scala.util.control.NonFatal
+
+
+object VerifyJanusData {
+  def main(args: Array[String]): Unit = {
+    args.headOption.fold {
```

Comment:
It occurs to me now, that a `match` might be easier to read for people that aren't used to `Option.fold` here. No biggie, I like `Option.fold` myself so this is totally fine as-is, up to you.